### ISSUE TITLE
Low Wattage PD Charger Connected Notification

### ISSUE THREAD

**Issue by JSandhu03** [seerge/g-helper#799]
So, I have the Zephyrus G14 2020 (4800HS, 1660TI) and with Armory Crate when I used to connect a slower USB-C charger (iPad charger), I would get a visual OSD alert for PD charging and a notification saying that the charger is slow. This was a nice indicator to check if my laptop was charging or not but I don't get this alert with G-Helper. I know this affects a very niche group of users, but would love to have this feature :)

**Comment by fruitlessfiddle** [comment]
Yeah I agree with you it would be nice if possible to see this. Could it also be added that you could see the wattage being charge. I know nothing of programing, so I apologize if I am suggesting impossible features.

**Comment by seerge** [comment]
@JSandhu03 @fruitlessfiddle I don't see any events from asus bios that would indicate a "slow" charger anyhow. Looking at charge rate itself - is not enough (as even a good 100W charger can start charting with 5W and then accelerate with time)

But I have added a Charge rate to the app (next to discharge), hope it helps :) 
[GHelper.zip](https://github.com/seerge/g-helper/files/12078459/GHelper.zip)

**Comment by fruitlessfiddle** [comment]
omg, thank you so much sir.

**Comment by fruitlessfiddle** [comment]
@seerge I dont know how to make it visible. I was checking but does this not show itself when battery is at a 100%.

**Comment by seerge** [comment]
@fruitlessfiddle discharge and charge again. When it's at 100% it's not charging anymore.
